index.html

```diff
@@ -118,12 +116,13 @@ <h2 class="multi_post_stories">Multi-Post Stories</h2>
         </div>
       </section>
 
-
       <!-- recent project 1 -->
       <div class="grid-container">
```

Comment:
- [ ] As demanded by project requirements and suggested by the previous reviewer, all work cards should be removed completely from the HTML and should be added dynamically using javascript. You should use an array of objects to store projects and render them to HTML on page load. You can use the code below as an example.
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}];

projects.forEach((project) => {
  const section = document.createElement('div');
  section.classList.add('work-card');
  /*Create all other necessary elements for the card and add appropriate classes*/
  workSection.appendChild(section);

  /*workSection should be selected from the html*/
});
```